--- sonata-1.5.3/setup.py.wiget	2008-09-14 06:45:42.000000000 +0200
+++ sonata-1.5.3/setup.py	2008-09-20 20:09:26.000000000 +0200
@@ -36,7 +36,7 @@ def rmgeneric(path, __func__):
 if not os.path.exists("mo/"):
 	os.mkdir("mo/")
 for lang in ('de', 'pl', 'ru', 'fr', 'zh_CN', 'sv', 'es', 'fi', 'uk', 'it', 'cs', \
-             'nl', 'pt_BR', 'da', 'be@latin', 'et', 'ca', 'ar', 'tr', 'el_GR'):
+             'nl', 'pt_BR', 'da', 'be@latin', 'et', 'ca', 'ar', 'tr', 'el'):
 	pofile = "po/" + lang + ".po"
 	mofile = "mo/" + lang + "/sonata.mo"
 	if not os.path.exists("mo/" + lang + "/"):
@@ -92,7 +92,7 @@ setup(name='Sonata',
                     ('share/locale/et/LC_MESSAGES', ['mo/et/sonata.mo']),
                     ('share/locale/tr/LC_MESSAGES', ['mo/tr/sonata.mo']),
                     ('share/locale/be@latin/LC_MESSAGES', ['mo/be@latin/sonata.mo']),
-                    ('share/locale/el_GR/LC_MESSAGES', ['mo/el_GR/sonata.mo']),
+                    ('share/locale/el/LC_MESSAGES', ['mo/el/sonata.mo']),
                     ('share/locale/uk/LC_MESSAGES', ['mo/uk/sonata.mo'])],
         )
 
