diff -burNp sonata-1.6.2/setup.py sonata-1.6.2-dud/setup.py
--- sonata-1.6.2/setup.py	2009-03-18 03:23:09.000000000 +0100
+++ sonata-1.6.2-dud/setup.py	2009-07-26 17:33:33.124429229 +0200
@@ -35,7 +35,7 @@ def rmgeneric(path, __func__):
 if not os.path.exists("mo/"):
 	os.mkdir("mo/")
 for lang in ('de', 'pl', 'ru', 'fr', 'zh_CN', 'sv', 'es', 'fi', 'uk', 'it', 'cs', \
-             'nl', 'pt_BR', 'da', 'be@latin', 'et', 'ca', 'ar', 'tr', 'el_GR', 'sk', \
+             'nl', 'pt_BR', 'da', 'be@latin', 'et', 'ca', 'ar', 'tr', 'el', 'sk', \
              'zh_TW', 'ja', 'sl'):
 	pofile = "po/" + lang + ".po"
 	mofile = "mo/" + lang + "/sonata.mo"
@@ -99,7 +99,7 @@ setup(name='Sonata',
                     ('share/locale/et/LC_MESSAGES', ['mo/et/sonata.mo']),
                     ('share/locale/tr/LC_MESSAGES', ['mo/tr/sonata.mo']),
                     ('share/locale/be@latin/LC_MESSAGES', ['mo/be@latin/sonata.mo']),
-                    ('share/locale/el_GR/LC_MESSAGES', ['mo/el_GR/sonata.mo']),
+                    ('share/locale/el/LC_MESSAGES', ['mo/el/sonata.mo']),
                     ('share/locale/sk/LC_MESSAGES', ['mo/sk/sonata.mo']),
                     ('share/locale/ja/LC_MESSAGES', ['mo/ja/sonata.mo']),
                     ('share/locale/sl/LC_MESSAGES', ['mo/sl/sonata.mo']),
